Ed-Fi-Alliance-OSS/Data-Management-Service
# [DMS-392] Configuration Service Release Management

**Steps to test:**
Execute:
```
git tag -a v0.3.X -m "v0.3.X"
git push origin v0.3.X
```

	In tag you will see:
		v0.3.X
		dms-pre-0.3.X
		cs-pre-0.3.X
	In Release:
		dms-v0.3.X
		cs-v0.3.X

**Additional Test:**
1. Do some change on DMS, create a PR an Merge it
	In tag you will see:
		dms-pre-0.3.X-alpha.0.X
	In Release:
		dms-v0.3.X-alpha.0.1
2. Do some change on Config, create a PR an Merge it
	In tag you will see:
		cs-pre-0.3.X-alpha.0.X
	In Release:
		cs-v0.3.X-alpha.0.1
